app.js

```diff
@@ -16,7 +16,18 @@ function Projects (title, client, author, published, description, tech, image){
 
```

Comment:
The constructor is a great start! Take a look at the article code, specifically that there's one `object